docs/userGuide/syntax/boxes.mbdf

```diff
@@ -160,6 +160,37 @@
 </span>
 </include>
 
+**You can also use icons and resize them accordingly.**
+
+<include src="outputBox.md" boilerplate >
+<span id="code">
+```html
+<box type="warning" icon=":fas-camera:" icon-size="2x" heading="Tip box heading">
```

Comment:
Can we have the same tipbox type with the same content for all three tipboxes? This would make it easier to see the contrast in size (both visually and in code), and I don't think we need the `heading` or `dismissible` attributes for these examples.

The size showcased should be no-resize, "2x" and "3x".